### ISSUE TITLE
Card mechanics explanations

### ISSUE THREAD

**Issue by jacwah** [Cardshifter/HTML-Client#91]
In addition to visual cues (see #74) I really like the hover-over effect explanations in HS. For instance: if a card has Taunt and you hover over it, a more detailed description of the mechanic is shown.

This is really useful for someone learning the game without having to go through and remember extensive tutorials or read a reference guide.

![HearthStone screenshot](http://i.imgur.com/UNPosWG.jpg)

**Comment by Zomis** [comment]
Indeed, I especially thing the Ranged feature could use some in-game explanation, as in addition to preventing return damage, it also only allows the creature to attack each 2nd turn.

In addition, the mana upkeep feature in Mythos could use some in-game explanation.

IMO, having in-game explanations is better than expecting users to read "game rules" documents.

**Comment by Phrancis** [comment]
I don't mind "floating" hints like that, probably pretty easy to do the CSS for it, if you guys can get the Angular part working I'll volunteer for the HTML/CSS.

**Comment by Phrancis** [comment]
This may be fairly easy to implement to `CardModel` in Vue.js client using Bootstrap popovers/tooltips.
